app/src/main/java/com/islaharper/dawnofandroid/presentation/screens/splash/SplashScreen.kt

```diff
@@ -71,7 +73,7 @@ private fun Splash(composition: LottieComposition?, progress: Float) {
             Spacer(modifier = Modifier.height(200.dp))
             Text(
                 text = stringResource(R.string.app_name),
-                color = MaterialTheme.colorScheme.onSecondary,
+                color = White,
```

Comment:
Why not use a theme color here? `White` is `onSecondary` in the theme

Reply:
I did have it using tertiary and onTertiary etc, but unhappy with the way it looks, have created an issue, https://github.com/DurtDiv3r/Dawn-of-Android/issues/8 , to finalise the colourScheme at a later point.

Reply:
Theme colours updated and implemented.